5.process_CP_features/scripts/2.normalize_sc_across_time.py

```diff
@@ -34,14 +34,14 @@
 
 # dictionary with each run for the cell type
 dict_of_inputs = {
-    "run_20230920ChromaLiveTL_24hr4ch_MaxIP": {
-        "annotated_file_path": pathlib.Path(
-            f"{data_dir}/run_20230920ChromaLiveTL_24hr4ch_MaxIP_sc.parquet"
-        ).resolve(),
-        "output_file_path": pathlib.Path(
-            f"{output_dir}/run_20230920ChromaLiveTL_24hr4ch_MaxIP_norm.parquet"
-        ).resolve(),
-    },
+    # "run_20230920ChromaLiveTL_24hr4ch_MaxIP": {
+    #     "annotated_file_path": pathlib.Path(
+    #         f"{data_dir}/run_20230920ChromaLiveTL_24hr4ch_MaxIP_sc.parquet"
+    #     ).resolve(),
+    #     "output_file_path": pathlib.Path(
+    #         f"{output_dir}/run_20230920ChromaLiveTL_24hr4ch_MaxIP_norm.parquet"
+    #     ).resolve(),
+    # },
```

Comment:
Recommend removing commented out code if not needed.

Reply:
removed